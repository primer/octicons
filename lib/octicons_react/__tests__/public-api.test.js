import * as Octicons from '../'

describe('@primer/octicons-react', () => {
  it('should not update exports without a semver change', () => {
    expect(Object.keys(Octicons).sort()).toMatchSnapshot()
  })
})
