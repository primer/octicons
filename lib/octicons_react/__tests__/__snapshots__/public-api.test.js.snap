// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`@primer/octicons-react should not update exports without a semver change 1`] = `
[
  "AccessibilityIcon",
  "AccessibilityInsetIcon",
  "AlertFillIcon",
  "AlertIcon",
  "AppsIcon",
  "ArchiveIcon",
  "ArrowBothIcon",
  "ArrowDownIcon",
  "ArrowDownLeftIcon",
  "ArrowDownRightIcon",
  "ArrowLeftIcon",
  "ArrowRightIcon",
  "ArrowSwitchIcon",
  "ArrowUpIcon",
  "ArrowUpLeftIcon",
  "ArrowUpRightIcon",
  "BeakerIcon",
  "BellFillIcon",
  "BellIcon",
  "BellSlashIcon",
  "BlockedIcon",
  "BoldIcon",
  "BookIcon",
  "BookmarkFillIcon",
  "BookmarkIcon",
  "BookmarkSlashFillIcon",
  "BookmarkSlashIcon",
  "BriefcaseIcon",
  "BroadcastIcon",
  "BrowserIcon",
  "BugIcon",
  "CacheIcon",
  "CalendarIcon",
  "CheckCircleFillIcon",
  "CheckCircleIcon",
  "CheckIcon",
  "CheckboxIcon",
  "ChecklistIcon",
  "ChevronDownIcon",
  "ChevronLeftIcon",
  "ChevronRightIcon",
  "ChevronUpIcon",
  "CircleIcon",
  "CircleSlashIcon",
  "ClockIcon",
  "CloudIcon",
  "CloudOfflineIcon",
  "CodeIcon",
  "CodeOfConductIcon",
  "CodeReviewIcon",
  "CodeSquareIcon",
  "CodescanCheckmarkIcon",
  "CodescanIcon",
  "CodespacesIcon",
  "ColumnsIcon",
  "CommandPaletteIcon",
  "CommentDiscussionIcon",
  "CommentIcon",
  "CommitIcon",
  "ContainerIcon",
  "CopilotErrorIcon",
  "CopilotIcon",
  "CopilotWarningIcon",
  "CopyIcon",
  "CpuIcon",
  "CreditCardIcon",
  "CrossReferenceIcon",
  "DashIcon",
  "DatabaseIcon",
  "DependabotIcon",
  "DesktopDownloadIcon",
  "DeviceCameraIcon",
  "DeviceCameraVideoIcon",
  "DeviceDesktopIcon",
  "DeviceMobileIcon",
  "DiamondIcon",
  "DiffAddedIcon",
  "DiffIcon",
  "DiffIgnoredIcon",
  "DiffModifiedIcon",
  "DiffRemovedIcon",
  "DiffRenamedIcon",
  "DotFillIcon",
  "DotIcon",
  "DownloadIcon",
  "DuplicateIcon",
  "EllipsisIcon",
  "EyeClosedIcon",
  "EyeIcon",
  "FeedDiscussionIcon",
  "FeedForkedIcon",
  "FeedHeartIcon",
  "FeedMergedIcon",
  "FeedPersonIcon",
  "FeedRepoIcon",
  "FeedRocketIcon",
  "FeedStarIcon",
  "FeedTagIcon",
  "FeedTrophyIcon",
  "FileAddedIcon",
  "FileBadgeIcon",
  "FileBinaryIcon",
  "FileCodeIcon",
  "FileDiffIcon",
  "FileDirectoryFillIcon",
  "FileDirectoryIcon",
  "FileDirectoryOpenFillIcon",
  "FileIcon",
  "FileMediaIcon",
  "FileMovedIcon",
  "FileRemovedIcon",
  "FileSubmoduleIcon",
  "FileSymlinkFileIcon",
  "FileZipIcon",
  "FilterIcon",
  "FlameIcon",
  "FoldDownIcon",
  "FoldIcon",
  "FoldUpIcon",
  "GearIcon",
  "GiftIcon",
  "GitBranchIcon",
  "GitCommitIcon",
  "GitCompareIcon",
  "GitMergeIcon",
  "GitMergeQueueIcon",
  "GitPullRequestClosedIcon",
  "GitPullRequestDraftIcon",
  "GitPullRequestIcon",
  "GlobeIcon",
  "GrabberIcon",
  "GraphIcon",
  "HashIcon",
  "HeadingIcon",
  "HeartFillIcon",
  "HeartIcon",
  "HistoryIcon",
  "HomeFillIcon",
  "HomeIcon",
  "HorizontalRuleIcon",
  "HourglassIcon",
  "HubotIcon",
  "IdBadgeIcon",
  "ImageIcon",
  "InboxIcon",
  "InfinityIcon",
  "InfoIcon",
  "IssueClosedIcon",
  "IssueDraftIcon",
  "IssueOpenedIcon",
  "IssueReopenedIcon",
  "ItalicIcon",
  "IterationsIcon",
  "KebabHorizontalIcon",
  "KeyAsteriskIcon",
  "KeyIcon",
  "LawIcon",
  "LightBulbIcon",
  "LinkExternalIcon",
  "LinkIcon",
  "ListOrderedIcon",
  "ListUnorderedIcon",
  "LocationIcon",
  "LockIcon",
  "LogIcon",
  "LogoGistIcon",
  "LogoGithubIcon",
  "MailIcon",
  "MarkGithubIcon",
  "MarkdownIcon",
  "MegaphoneIcon",
  "MentionIcon",
  "MeterIcon",
  "MilestoneIcon",
  "MirrorIcon",
  "MoonIcon",
  "MortarBoardIcon",
  "MultiSelectIcon",
  "MuteIcon",
  "NoEntryFillIcon",
  "NoEntryIcon",
  "NorthStarIcon",
  "NoteIcon",
  "NumberIcon",
  "OrganizationIcon",
  "PackageDependenciesIcon",
  "PackageDependentsIcon",
  "PackageIcon",
  "PaintbrushIcon",
  "PaperAirplaneIcon",
  "PaperclipIcon",
  "PasteIcon",
  "PencilIcon",
  "PeopleIcon",
  "PersonAddIcon",
  "PersonFillIcon",
  "PersonIcon",
  "PinIcon",
  "PlayIcon",
  "PlugIcon",
  "PlusCircleIcon",
  "PlusIcon",
  "ProjectIcon",
  "PulseIcon",
  "QuestionIcon",
  "QuoteIcon",
  "ReplyIcon",
  "RepoCloneIcon",
  "RepoDeletedIcon",
  "RepoForkedIcon",
  "RepoIcon",
  "RepoLockedIcon",
  "RepoPullIcon",
  "RepoPushIcon",
  "RepoTemplateIcon",
  "ReportIcon",
  "RocketIcon",
  "RowsIcon",
  "RssIcon",
  "RubyIcon",
  "ScreenFullIcon",
  "ScreenNormalIcon",
  "SearchIcon",
  "ServerIcon",
  "ShareAndroidIcon",
  "ShareIcon",
  "ShieldCheckIcon",
  "ShieldIcon",
  "ShieldLockIcon",
  "ShieldSlashIcon",
  "ShieldXIcon",
  "SidebarCollapseIcon",
  "SidebarExpandIcon",
  "SignInIcon",
  "SignOutIcon",
  "SingleSelectIcon",
  "SkipIcon",
  "SlidersIcon",
  "SmileyIcon",
  "SortAscIcon",
  "SortDescIcon",
  "SquareFillIcon",
  "SquareIcon",
  "SquirrelIcon",
  "StackIcon",
  "StarFillIcon",
  "StarIcon",
  "StopIcon",
  "StopwatchIcon",
  "StrikethroughIcon",
  "SunIcon",
  "SyncIcon",
  "TabExternalIcon",
  "TabIcon",
  "TableIcon",
  "TagIcon",
  "TasklistIcon",
  "TelescopeFillIcon",
  "TelescopeIcon",
  "TerminalIcon",
  "ThreeBarsIcon",
  "ThumbsdownIcon",
  "ThumbsupIcon",
  "ToolsIcon",
  "TrashIcon",
  "TriangleDownIcon",
  "TriangleLeftIcon",
  "TriangleRightIcon",
  "TriangleUpIcon",
  "TrophyIcon",
  "TypographyIcon",
  "UnfoldIcon",
  "UnlockIcon",
  "UnmuteIcon",
  "UnverifiedIcon",
  "UploadIcon",
  "VerifiedIcon",
  "VersionsIcon",
  "VideoIcon",
  "WebhookIcon",
  "WorkflowIcon",
  "XCircleFillIcon",
  "XCircleIcon",
  "XIcon",
  "ZapIcon",
  "default",
]
`;
